tests/tasks/test_notebook.py

```diff
@@ -1383,3 +1383,31 @@ def test_notebook_add_parameters_cell_if_missing(
         else jupytext.guess_format(path.read_text(), ext=path.suffix)
     )
     assert fmt == expected_fmt
+
+
+def test_local_import_ScriptRunner(tmp_directory, capsys):
```

Comment:
let's add one more tests where PYTHONPATH is added to ensure we're not overwriting it